README.md

```diff
@@ -43,6 +43,10 @@ Wraps module into an ewasm-compatible constructor. It has two presets:
 - `memory`: wrap the module as a pre-defined memory section
 - `customsection`: include the module as a custom section
 
+### repack
+
+Re-serializes the module. It will drop any unknown (custom) sections.
```

Comment:
I think the wording in the first sentence is a bit unintuitive. Maybe the second sentence on its own will do.

Reply:
Well the second sentence is not only what it does, it is a side effect.